discovery-infra/test_infra/consts/consts.py

```diff
@@ -47,6 +47,22 @@
 DEFAULT_TEST_INFRA_DOMAIN = f".{CLUSTER_PREFIX}-{DEFAULT_NAMESPACE}.{DEFAULT_BASE_DNS_DOMAIN}"
 TEST_TARGET_INTERFACE = "vnet3"
 SUFFIX_LENGTH = 8
+NODES_DEPLOYMENT_LOCK_FILE = "/tmp/nodes_deployment.lock"
+
+REQUIRED_ASSET_FIELDS = (
```

Comment:
That not what I meant.
You are writing `machine_cidr` `machine_cidr6` and more for both `REQUIRED_ASSET_FIELDS` and `IP_NETWORK_ASSET_FIELDS` (and probably other places on test infra)
We might want them to be on class/module and create `REQUIRED_ASSET_FIELDS` using that class/module.
Later on we will be able to use those consts on all over the project.